x-pack/plugins/infra/public/containers/logs/log_analysis/log_analysis_cleanup.tsx

```diff
@@ -46,8 +46,10 @@ const waitUntilJobsAreDeleted = async <JobType extends string>(
 ) => {
   const moduleJobIds = jobTypes.map((jobType) => getJobId(spaceId, logViewId, jobType));
   while (true) {
```

Comment:
could this be an infinite loop if there's an issue with the api ? a timeout could be useful as another exit condition

Reply:
Yeah, good point, let me add that!